### ISSUE TITLE
Revise search endpoint

### ISSUE THREAD

**Issue by wdprice** [topcoder-platform/u-bahn-app#640]
v5/search should not be abstracted at this level.

should be app specific (ie: v5/skill-search/)

Update all references to endpoint to use new one

**Comment by callmekatootie** [comment]
@cwdcwd Not sure I understood this right - we replace `/v5/search` endpoints with `/v5/skill-search` instead? Just the path name changes?

**Comment by cwdcwd** [comment]
yes I think for now that's what we should do. `search` as top level "noun" is somewhat ambiguous and pollutes the namespace somewhat

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30141385 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30141385 has been updated - it has been assigned to callmekatootie.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30141385<br/><br/>```This is an automated message for lazybaer via Topcoder X```
